addons/l10n_in_ewaybill_stock/models/l10n_in_ewaybill.py

```diff
@@ -436,6 +436,11 @@ def _generate_ewaybill_direct(self):
         self._l10n_in_ewaybill_stock_handle_zero_distance_alert_if_present(response)
         self._write_successfully_response(response_values)
         self._cr.commit()
+        pdf_content = self.env["ir.actions.report"]._render_qweb_pdf("l10n_in_ewaybill_stock.action_report_ewaybill", res_ids=[self.id])[0]
```

Comment:
```suggestion
        pdf_content = self.env['ir.actions.report']._render_qweb_pdf(
            'l10n_in_ewaybill_stock.action_report_ewaybill', res_ids=self.ids)[0]
```
1. Use single quotes(`''`) for technical strings
2. Line exceeds 120 char makes it hard to in a normal editor (Usually it's better to follow PEP-8 Guidelines)